web/api/vcap-local.template.json

```diff
@@ -19,6 +19,13 @@
         "discoveryEndpoint": "",
         "scopes": []
       }
+    },
+    "Cloudant_sensors": {
```

Comment:
Since the vcap-local file is being updated, can you update the readme to include instructions on the new fields?

Reply:
Yes, I'll add that to the README -- I'll also add it to the cf manifest file, to keep that deployment alive for the time being.